sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionReceiverClient.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * The builder that creates session-related {@link ServiceBusReceiverClient} instances.
+ * Use {@link #acceptSession(String)} to create a {@link ServiceBusReceiverClient} that is tied to a known specific
+ * session id.
+ * Use {@link #acceptNextSession()} to create a {@link ServiceBusReceiverClient} that is tied to an unknown
+ * available session.
+ * Use {@link #getReceiverClient(int)} to create a {@link ServiceBusReceiverClient} that
+ * process events from up to maxConcurrentSessions number of sessions.
+ */
+public class ServiceBusSessionReceiverClient implements AutoCloseable {
+    private final ServiceBusSessionReceiverAsyncClient sessionAsyncClient;
+    private final Duration operationTimeout;
+
+    ServiceBusSessionReceiverClient(ServiceBusSessionReceiverAsyncClient asyncClient, Duration operationTimeout) {
+        this.sessionAsyncClient = Objects.requireNonNull(asyncClient, "'asyncClient' cannot be null.");
+        this.operationTimeout = Objects.requireNonNull(operationTimeout, "'operationTimeout' cannot be null.");
```

Comment:
Why do we need `operationTimeout` as ctor arg. We can include this in the `accept*Session` APIs.

Reply:
According to our latest offline discussion, we'll keep this operationTimeout on client.